src/types/inc/Utf8Utf16Convert.hpp

```diff
@@ -9,74 +9,83 @@ Module Name:
 - Defines classes which hold the status of the current partials handling.
 - Defines functions for converting between UTF-8 and UTF-16 strings.
 
+Tests have been made in order to investigate whether or not own algorithms
+could overcome disadvantages of syscalls. Test results can be read up
+in PR #4093 and the test algorithms are available in src\tools\U8U16Test.
+Based on the results the decision was made to keep using the platform
+functions MultiByteToWideChar and WideCharToMultiByte.
+
 Author(s):
 - Steffen Illhardt (german-one) 2019
 --*/
 
 #pragma once
 
-class u8state final
+namespace til // Terminal Implementation Library. Also: "Today I Learned"

```

Comment:
Can we also move the file location to ./src/inc/til/?

Reply:
Oooh... Maybe not because it has a .cpp implementation. @DHowett-MSFT, what did we say we were going to do about things like this?